game_eggs/steamcmd_servers/palworld/egg-palworld.json

```diff
@@ -17,7 +17,7 @@
     "file_denylist": [
         "PalServer.sh"
     ],
-    "startup": ".\/PalworldServerConfigParser; (while read cmd; do rcon -s -a \"localhost:$RCON_PORT\" -p \"$ADMIN_PASSWORD\" \"$cmd\";done) < \/dev\/stdin & \/home\/container\/Pal\/Binaries\/Linux\/PalServer-Linux-Test Pal EpicApp=PalServer -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS -publicip={{PUBLIC_IP}} -port={{SERVER_PORT}} -publicport={{SERVER_PORT}} -servername=\"{{SERVER_NAME}}\" -players={{MAX_PLAYERS}} $(if [ -n \"$SERVER_PASSWORD\" ]; then echo \"-serverpassword=\\\"${SERVER_PASSWORD}\\\"\"; fi) -adminpassword=\"{{ADMIN_PASSWORD}}\"",
+    "startup": ".\/PalworldServerConfigParser; (while read cmd; do rcon -s -a \"localhost:$RCON_PORT\" -p \"$ADMIN_PASSWORD\" \"$cmd\";done) < \/dev\/stdin & \/home\/container\/Pal\/Binaries\/Linux\/PalServer-Linux-Test Pal EpicApp=PalServer -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS -publicip={{PUBLIC_IP}} -port={{SERVER_PORT}} -publicport={{SERVER_PORT}} -servername=\"{{SERVER_NAME}}\" -players={{MAX_PLAYERS}} $(if [ -n \"$SERVER_PASSWORD\" ]; then echo \"-serverpassword=\\\"${SERVER_PASSWORD}\\\"\"; fi) -adminpassword=\"{{ADMIN_PASSWORD}}\" -rcon",
```

Comment:
Please edit the startup line as the following.

``./PalworldServerConfigParser; (while read cmd; do rcon -s -a "localhost:$RCON_PORT" -p "$ADMIN_PASSWORD" "$cmd";done) < /dev/stdin & /home/container/Pal/Binaries/Linux/PalServer-Linux-Test Pal -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS -publiclobby -publicip={{PUBLIC_IP}} -port={{SERVER_PORT}} -publicport={{SERVER_PORT}} -servername="{{SERVER_NAME}}" -players={{MAX_PLAYERS}} $(if [ -n "$SERVER_PASSWORD" ]; then echo "-serverpassword=\"${SERVER_PASSWORD}\""; fi) -adminpassword="{{ADMIN_PASSWORD}}" -rcon``

They have removed EpicApp=PalServer and changed it to ``-publiclobby``

Can be found in the docs: https://tech.palworldgame.com/settings-and-operation/arguments

Reply:
after updating this the server will successfully show up in the revamped community servers tab :)

Reply:
When reading this comment it says the opposite of tis no?
Removing it = fixing it

However they fully removed this command line with the patch and replaced it with -publiclobby.
If this has an impact we will see i guess.
But ofc. it can also be a boolean option to toggle on/off. I'd be fine with that if Quinten also goes with it.

Sidenote: I know it is not your fault and i just see it right now and also dont know where this comes from but in the startcommand there is a lonely ``Pal`` right in front of the -publiclobby parameter :D This doesnt seem to be correct there ^^

Reply:
@Ballaual You updated the config parser with this new command ?
What I should do now ? 👀

Reply:
The changes to the parser needs to be approved and merged by Quinten first.

The new command is bShowPlayerList but isnt to br implemented inzo thr base egg.

Pls just remove the Pal before the -publiclobby , so the egg is clean. My fault i didnt remove it above -.- Sorryyyy..

After fixing this we just can  wait for Quinten to approve the changes and givr us ferdback on the toggle option above.

Reply:
> @Ballaual You updated the config parser with this new var `PUBLIC_SERVER` ? What I should do now ? 👀

nothing to do for this PR. Its just a variable for custom eggs if someone wants to add this to their egg options